### ISSUE TITLE
Make "Classic Phrasing" the default phrasing on Musesounds orchestral instruments.

### ISSUE THREAD

**Issue by Valdessir** [musescore/MuseScore#22631]
### Your idea

The "Classic Phrasing" option found in the sound flags of Musebrass instruments should become the default phrasing model of all orchestral Musesound instruments (woodwinds, brass, and strings). Generally, these instruments should distinctly rearticulate the notes on the score, and play legato when notes are inside a slur marking.
 
The behavior of the playback in classical phrasing, where notes are played detached but the notes inside the slur are played legato, is seen as an ideal default for the rest of the sections of the orchestra. The current default phrasing, which is labeled as "Standard," should then be relabeled as "Legato." Repurposing "Standard" into "Legato" is more fitting since it is the desired effect when there is "Legato" written on the score. 

### Problem to be solved

There has been some feedback with the current default phrasing model of Musesounds since its release. Its sempre legato phrasing, though graceful and beautiful to the ears, can be out of place when the scenario does not call for legato. It can, to some extent, result in notes being mumbled due to fast pieces that call for snappier note attacks.

#15099
#17602
#15202
#22140
#14464
#14524
#13878
#20701

### Prior art

https://github.com/musescore/MuseScore/assets/102687285/a3f14f82-790e-4ad1-bdee-722a5b1f8e09

Attached here is a concept video demonstrating a short arrangement applying this concept. See how the woodwinds and strings play legato inside the slur and play detached when there isn't, resulting in a nuanced and more realistic playback.

### Additional context

Having classic phrasing be the default phrasing for woodwinds, brass, and strings would improve the realism of Musesounds playback since orchestral performers in real life play notes detached by default, and play legato when marked with a slur or the legato staff text.

**Comment by wizofaus** [comment]
"Generally, these instruments should play the notes on the score with a clear separation between the notes" - I would dispute that.  Without slurs notes should be *rearticulated* (i.e. tongued/change of bow etc.) but should still be held for the full duration, unless otherwise specified.

**Comment by Valdessir** [comment]
Thanks for the feedback. I think that's a better word than saying "clear separation." English is not my first language.

**Comment by xbergade** [comment]
I agree, at the very least for the (solo) version of the strings Muse Sounds.
